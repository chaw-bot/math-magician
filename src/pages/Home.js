import React from 'react';

const Home = () => {
  const texts = {
    heading: 'Welcome to our page!',
    paragraph1: 'Lorem ipsum dolor sit amet,consectetur adipiscing elit. Pellentesque tempor ac nisi et facilisis. Vestibulum imperdiet, felis vitae lobortis porta,erat libero tempus arcu, id cursus lectus velit arcu. Phasellus venenatis venenatis elementum.Quisque tristique ipsum ac dui lobortis semper.',
    paragraph2: 'Duis placerat orci ultrices, elementum sem ut, tempus enim. Aenean in nulla mauris. Proin varius efficitur risus, eget gravida est viverra eget. Aenean eleifend laoreet tempor. Donec varius venenatis quam vel rutrum. Aliquam orci nulla, feugiat in dignissim at, ullamcorper ac metus.',
  };

  return (
    <div className="home-details">
      <h2 className="home-title">{texts.heading}</h2>
      <div className="home-para">
        {texts.paragraph1}
        <br />
        <br />
        {texts.paragraph2}
      </div>
    </div>
  );
};

export default Home;
