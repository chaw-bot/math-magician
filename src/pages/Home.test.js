import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('Render Home page', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});