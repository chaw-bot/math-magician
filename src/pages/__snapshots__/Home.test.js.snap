// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Home page 1`] = `
<div
  className="home-details"
>
  <h2
    className="home-title"
  >
    Welcome to our page!
  </h2>
  <div
    className="home-para"
  >
    Lorem ipsum dolor sit amet,consectetur adipiscing elit. Pellentesque tempor ac nisi et facilisis. Vestibulum imperdiet, felis vitae lobortis porta,erat libero tempus arcu, id cursus lectus velit arcu. Phasellus venenatis venenatis elementum.Quisque tristique ipsum ac dui lobortis semper.
    <br />
    <br />
    Duis placerat orci ultrices, elementum sem ut, tempus enim. Aenean in nulla mauris. Proin varius efficitur risus, eget gravida est viverra eget. Aenean eleifend laoreet tempor. Donec varius venenatis quam vel rutrum. Aliquam orci nulla, feugiat in dignissim at, ullamcorper ac metus.
  </div>
</div>
`;
